lighthouse-core/report/templates/report-template.html

```diff
@@ -177,3 +176,13 @@ <h1 class="menu__header-title">Lighthouse</h1>
     </footer>
   </section>
 </div>
+
+{{#if scripts }}
+<script>
+  window.lhresults = {{{lhresults}}};
```

Comment:
would it be totally crazy to do even more defense in depth here by putting `lhresults` in an html element instead of in a `<script>`?

Really we should [be doing more](https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet#RULE_.233_-_JavaScript_Escape_Before_Inserting_Untrusted_Data_into_JavaScript_Data_Values) to escape our JS, but one end run around this is to do double stash `{{lhresults}}` in a (display: none) div, so we get HTML entity escaping, then get the actual results when needed by `JSON.parse(lhresultsDiv.textContent)` (see [rule 1](https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet#RULE_.231_-_HTML_Escape_Before_Inserting_Untrusted_Data_into_HTML_Element_Content) and the summary section for more details)